.github/ISSUE_TEMPLATE/release_a_new_version.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Release a New Version!
+about: Prepare a release to be launched
+title: ''
+labels: release
+---
+
+<!-- Please Follow this checklist before making your release. Thanks! -->
+
+### Release Checklist
+
+> Attention: the examples below use the version `0.1.0` but you should update them to use the version you're releasing.
+
+- [ ] Decide on a version number based on the current version number and the common rules defined in [Semantic Versioning](https://semver.org). E.g. `0.1.0`.
+- [ ] Update this ticket name to reflect the new version number, following the pattern "Release `0.1.0`".
+- [ ] Cut a branch for the new release out of the `develop` branch, following the gitflow naming pattern `release/0.1.0`.
+- [ ] Update the code to use this version number.
+- [ ] Update the [CHANGELOG.md] file with the new version number and release notes.
+- [ ] Run tests and linting. **Not only CI/CD tests, but also manual tests to make sure the release is up and running, and that it's stable!**
+- [ ] Make all changes necessary to make sure the release is ready to be published. If new issues are found on manual tests, improve the CI so these issues can be automatically detected next time.
```

Comment:
We should definitely ensure that tests pass, but I don't know if we want to block releases on adding tests to the CI. What if we suggest filing an issue here instead?

Reply:
Good point, I'll update it to create new issues to automate these tests.